GeoLib/PointVec.h

```diff
@@ -103,6 +103,7 @@ class PointVec final : public TemplateVec<Point>
 
     void setNameForElement(std::size_t id, std::string const& name) override;
 
+    void resetInternalDataStructures();
```

Comment:
Please add a comment what is rebuild and if something is invalidated after this function (like pointers...).